[docs] added steps for docker install for airgapped (#16629)
yugabyte/yugabyte-db: docs/content/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/default.md

- CentOS (default)
- Ubuntu 18 and 20, via Replicated

> - Other [requirements for installation by Replicated](https://docs.replicated.com/enterprise/installing-general-requirements).

Reviewer: ```suggestion
Refer also to the other [requirements for installation via Replicated](https://docs.replicated.com/enterprise/installing-general-requirements).
```

> Refer also to the other [requirements for installation via Replicated](https://docs.replicated.com/enterprise/installing-general-requirements).

Reviewer: the list of OSs has since been updated, drop these changes and get what's on master

- 100 GB SSD disk or more
- x86-64 CPU architecture

> ### Kubernetes-based installations

Reviewer: there is a kubernetes tab for this

        ```

    1. Start Docker using the following command:

        ```sh
        sudo systemctl start docker.service
        ```

- Ensuring that the following ports are open on the YugabyteDB Anywhere host:
  - `8800` – HTTP access to the Replicated UI
  - `80` – HTTP access to the YugabyteDB Anywhere UI
  - `22` – SSH
- Ensuring that the attached disk storage (such as persistent EBS volumes on AWS) is 100 GB minimum.
- Having YugabyteDB Anywhere airgapped install package. Contact Yugabyte Support for more information.
- Signing the Yugabyte license agreement. Contact Yugabyte Support for more information.

### Kubernetes-based installations

The YugabyteDB Anywhere Helm chart has been tested using the following software versions:

- Kubernetes 1.22
- Helm 3.10

Before installing YugabyteDB Anywhere, verify that you have the following:

- A Kubernetes cluster with nodes configured according to the [hardware requirements](#kubernetes-based-installations).
- A client environment with the kubectl and Helm command-line tools configured with a service account or user that has admin access to a  single namespace on the subject Kubernetes cluster.

> - A Kubernetes secret obtained from [Yugabyte](https://www.yugabyte.com/platform/#request-trial-form).

Reviewer: use the YB platform support shortcode

In addition, ensure the following:

> - The host can pull container images from the [quay.io](https://quay.io/) container registry. If the host cannot do this, you need to prepare these images in your internal registry by following instructions provided in [Pull and push YugabyteDB Docker images to private container registry](../prepare-environment/kubernetes#pull-and-push-yugabytedb-docker-images-to-private-container-registry).

Reviewer: broken link

>   `https://downloads.yugabyte.com`

Reviewer: Remove the "Installing on airgapped hosts requires additional configurations..." sentence above, the section is right here.
Change first sentence "Installing YugabyteDB Anywhere on airgapped hosts (without access to any Internet traffic (inbound or outbound)) requires the following additional configuration:"

  `https://download.docker.com`

- Ensuring that Docker Engine version 20.10.n is available. If it is not installed, you need to follow the procedure described in [Installing Docker in airgapped](https://community.replicated.com/t/installing-docker-in-airgapped-environments/81).

    If you want to set up Docker on Amazon Linux OS, perform the following:

    1. Install Docker using the following commands:

        ```sh
        sudo yum install docker
        sudo systemctl daemon-reload
        ```

    1. Start Docker using the following command:

        ```sh
        sudo systemctl start docker.service
        ```

> - Ensuring that the following ports are open on the YugabyteDB Anywhere host:

Reviewer: ```suggestion
- Ensure that the following ports are open on the YugabyteDB Anywhere host:
```

> - Ensuring that the following ports are open on the YugabyteDB Anywhere host:

Author: > Are the rest of these bullets really only for airgapped? except for the airgapped package, they look like they apply to any install. why is storage here 100GB and 200GB above?

@iSignal , Can you please check this page for its accuracy as there has been some revamp to the Prerequisite section recently. So we want to make sure that the details on this page are valid. Here's the deploy preview for the page : https://deploy-preview-16629--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/default/

> - Ensuring that the following ports are open on the YugabyteDB Anywhere host:

Reviewer 2: 1. The bullet point for `Ensuring that the attached disk storage (such as persistent EBS volumes on AWS) is 100 GB minimum.` in this section - https://deploy-preview-16629--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/prerequisites/default/#airgapped-hosts -  could be removed because we state prereqs = 200 GB upfront.

Other than that, the prereqs page looks good.

        sudo systemctl start docker.service
        ```

- Ensure that the following ports are open on the YugabyteDB Anywhere host:
  - `8800` – HTTP access to the Replicated UI
  - `80` – HTTP access to the YugabyteDB Anywhere UI
  - `22` – SSH

> - Ensuring that the attached disk storage (such as persistent EBS volumes on AWS) is 100 GB minimum.

Reviewer: ```suggestion
- Ensure that the attached disk storage (such as persistent EBS volumes on AWS) is 100 GB minimum.
```

- Having YugabyteDB Anywhere airgapped install package. Contact {{% support-platform %}} for more information.

> - Signing the Yugabyte license agreement. Contact {{% support-platform %}} for more information.

Reviewer: ```suggestion
- Sign the Yugabyte license agreement. Contact {{% support-platform %}} for more information.
```
